.github/workflows/create-pr.yml

```diff
@@ -0,0 +1,43 @@
+name: Automatic PR for Development Update
+
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  create-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Extract version information
+        run: |
+          APP_VERSION=$(echo "${{ github.event.release.tag_name }}" | grep -oP '\d+\.\d+\.\d+')
+          echo "Extracted APP_VERSION: $APP_VERSION"
+          echo "APP_VERSION=$APP_VERSION" >> $GITHUB_ENV
+
+          VERSION=$APP_VERSION  
+          echo "Extracted VERSION: $VERSION"
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+
```

Comment:
This block is completely unnecessary since you can just re-use the variable created above.